lib/steps.js

```diff
@@ -731,7 +737,21 @@ After(async (scenario) => {
   }
 });
 
-AfterAll(async () => {
+AfterAll(async function() {
+  // Create report
+  let report = `# ALKiln REPORT - ${ (new Date()).toUTCString() }\n\n`
```

Comment:
What does `ALKiln` stand for? I'm blanking on that right now.

Reply:
lol, sorry, I don't think I mentioned this to anyone here - we're looking at reorganizing the repos that have to do with the Assembly Line project. We're planning to call this one the 'kiln', prefixed by 'AL' for Assembly Line. We can discuss that and change it if needed.